### ISSUE TITLE
Stuck at Writting password... for 2 days

### ISSUE THREAD

**Issue by Greendrift** [charlesbel/Microsoft-Rewards-Farmer#385]
### Before submitting a bug report...

- [X] This bug wasn't already reported.
(I have checked every bug report on GitHub)

### Title

- [X] The title is no longer "[BUG] Title" and I edited it with the right error name.

### Describe the bug

tittle, everything was working properly until yesterday

### Copy and paste your error

2023-10-23 13:16:56,092 [INFO] Using default request storage
2023-10-23 13:16:56,107 [INFO] Created proxy listening on 127.0.0.1:50356
2023-10-23 13:16:56,108 [INFO] Using undetected_chromedriver
2023-10-23 13:16:56,647 [INFO] patching driver executable C:\Users\green\appdata\roaming\undetected_chromedriver\undetected_chromedriver.exe
2023-10-23 13:16:57,812 [INFO] Screen size: 1431x1180
2023-10-23 13:16:57,812 [INFO] Device size: 1376x1029
2023-10-23 13:16:57,832 [INFO] [LOGIN] Logging-in...
2023-10-23 13:17:00,874 [INFO] [LOGIN] Writing email...
2023-10-23 13:17:02,524 [INFO] [LOGIN] Writing password...

### Screenshots

![image](https://github.com/charlesbel/Microsoft-Rewards-Farmer/assets/132684364/9a9e8b01-f141-4c5a-90e5-457fc6ac3b28)

### Value of dashboard variable

https://gist.github.com/Greendrift/8a7cebbe61cce4c5b04e54ac4fda5d81

**Comment by Lectoid** [comment]
This happens to me sometimes, I just Ctrl-C, kill any remaining chrome processes and then start the script again, usually completes the second time through. Rarely I have to do it again after that.

**Comment by bruno-lambreht** [comment]
If you run using "-v" argument, you gonna see "https://login.live.com/" it's not loading. It looks a MS Problem. It's happening now for me

**Comment by Ventec247** [comment]
It's also happening off and on to me too days at a time login.live.com will just hang after the writing password or after the logging in stage (does it in normal chrome too)

Other links like login.microsoft.com etc. continue to work, is it possible anyone in the know can modify the bot with alternative login URL's given this seems to be a common issue?

**Comment by idieli** [comment]
I was able to fix this for myself by making some edits to `src/login.py`. I noticed that when I ran the script with the `-v` option it was successfully logging in with my username and password but then stalling when it loaded `https://rewards.bing.com/`. All I did was change the hostname check on line 73 in `src/login.py` from 
`== "account.microsoft.com"` to `in ("account.microsoft.com", "rewards.bing.com")`. 

This caused the call to `self.utils.waitUntilVisible` on line 78 to fail but I just replaced it with `time.sleep(10)` since it times out after 10 seconds anyway. Definitely a quick and dirty fix but it seems to do the job.

**Comment by Natriumpikant** [comment]
> I was able to fix this for myself by making some edits to `src/login.py`. I noticed that when I ran the script with the `-v` option it was successfully logging in with my username and password but then stalling when it loaded `https://rewards.bing.com/`. All I did was change the hostname check on line 73 in `src/login.py` from `== "account.microsoft.com"` to `in ("account.microsoft.com", "rewards.bing.com")`.
> 
> This caused the call to `self.utils.waitUntilVisible` on line 78 to fail but I just replaced it with `time.sleep(10)` since it times out after 10 seconds anyway. Definitely a quick and dirty fix but it seems to do the job.

May you share your full code of `src/login.py-` cant reproduce it at my side. Thanks

**Comment by icescoop12** [comment]
> > I was able to fix this for myself by making some edits to `src/login.py`. I noticed that when I ran the script with the `-v` option it was successfully logging in with my username and password but then stalling when it loaded `https://rewards.bing.com/`. All I did was change the hostname check on line 73 in `src/login.py` from `== "account.microsoft.com"` to `in ("account.microsoft.com", "rewards.bing.com")`.
> > This caused the call to `self.utils.waitUntilVisible` on line 78 to fail but I just replaced it with `time.sleep(10)` since it times out after 10 seconds anyway. Definitely a quick and dirty fix but it seems to do the job.
> 
> May you share your full code of `src/login.py-` cant reproduce it at my side. Thanks

yes 2nd that as I'm getting the failed when trying to login at certain runs
please share your login.py file

**Comment by idieli** [comment]
@Natriumpikant @icescoop12 
I created a pull request for klept0's fork: https://github.com/klept0/MS-Rewards-Farmer/pull/42. You can either implement those changes yourself or wait until he confirms them.
If you're still having issues I'd recommend killing chrome and deleting the contents of the session folder before each run.

**Comment by ridai260** [comment]
Mine gets stuck loading after "Stay signed in?". Tried killing chrome and deleting session folder.
Can't navigate to any other page from that first tab either, its simply stuck loading forever.
